GOVERNANCE.md

```diff
@@ -175,39 +195,108 @@ The project Steward is currently: **@FredKSchott**
+### Project Teams
+
+Besides our contributor levels described above, there are additional roles and teams available that community members are welcome to join. Roles are a great way to organize around different projects and initiatives in our community. For example:
 
-Staff is a special designation for employees of [The Astro Technology Company](https://astro.build/company) that lives outside of our Governance model. The staff role was designed to help those of us working full-time on Astro to work productively without "skipping the line" and circumventing our governance model entirely.
+- `@team-docs` runs the `#docs` channel and organizes the growth and development of Astro documentation.
+- `@i18n-gang` runs the `#docs-i18n` channel and organizes translations in several languages.
+- `@support-squad` runs the `#support-threads` channel and helps anyone who needs help using Astro.
 
-The staff role was designed to offer instant **visibility** and **trust**, but not instant **authority.**
+Many of these team roles can be be browsed and joined automatically by visiting the `#manage-roles` channel in our Discord. Getting involved with a team is a great way to start contributing to Astro!
+
+### Moderator
+
+**Moderator** is a special role available to Maintainers (L2 and above). While all maintainers are granted permissions to moderate for bad behavior across our community, a Moderator actively takes on this the responsibility. For example, a community member may ping moderators (via the `@mods` role) to resolve spam posts or Code of Conduct violations.
+
+Trivial tasks (like removing spam) can be acted on unilaterally by a Moderator. Other non-trivial tasks (like assisting with or resolving a Code of Conduct violation) should involve the entire Moderator team (and in some cases, the project Steward).
 
 #### Privileges
 
-All privileges of the [Core Maintainer role](#level-3---core-mainainer), except...
+- `@mods` role on [Discord](https://astro.build/chat)
+- Invitation to the private `#moderators` channel on Discord
+- Invitation to the `staff` team on GitHub.
+
+#### Nomination
 
-- Instead of gaining Discord contributor roles (`@contributor`, `@maintainer`, `@core`) you will receive a special `@staff` role in Discord and GitHub that grants equivalent visibility and permissions as `@core`.
-- No voting abilities for the first 3 months of staff membership. Then, the role grants equivalent voting permissions as `@core` (see [Voting](#voting) below).
-- Not eligible for additional contributor levels while acting as `@staff`. You can retain all current contributor levels and can request a new nomination upon leaving `@staff` (See [Leaving Staff](#leaving-staff) below). This rule is designed to prevent a team of contributors/maintainers that is overwhelmingly ex-staff members.
+Any Maintainer (L2 and above) can self-nominate by messaging the project Steward (`@steward`) on Discord.
 
-#### Responsibilities
+### Technical Steering Committee (TSC)
```

Comment:
So does "member of the technical steering committee" end up being like a "super core member" in a sense, requiring expertise in the code base?

I'm sorry, I hadn't seen the reason for this change

Reply:
The reason is so that people who are heavily involved but not working on the codebase can still become part of core. So the TSC becomes what the current core sort of was; for people working on the Astro codebase and who can guide the technical direction of the project.

Reply:
yup, less "super core member" and more "one team of many". It is the only team right now with extra voting permissions in our goverance docs, but this aligns `tsc` closer to a project like `docs` or `support-squad`. 

It also opens the door for other teams to adopt their own special governance down the road, similar to TSC. For example, if there ever was a reason to add a voting mechanism for the docs team, we could define the docs voting system in this same place.

Reply:
hmm ok so trying to go in the direction of working groups, kinda?

tsc = team focused on the codebase
docs = team focused on the docs
support-squad = team focused on providing support in discord & github & more